cmd/podman/rm.go

```diff
@@ -42,6 +46,20 @@ Running containers will not be removed without the -f option.
 	}
 )
 
+// Attempt to remove containers
+func removeContainersFromStorage(runtime *libpod.Runtime, ctrs []string) {
+	storageService := runtime.StorageService()
```

Comment:
If we really want to do this, let's do it in libpod, so we don't need to expose these things